scikit_mol/transformers.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, minPath:int = 1, maxPath:int =7, useHs:bool = True, branchedP
         self.fpSize = fpSize
         self.numBitsPerFeature = numBitsPerFeature
         self.atomInvariantsGenerator = atomInvariantsGenerator
+        self.nBits = fpSize  # FIXME: the argument naming should be consistent and only use nBits
```

Comment:
I think we should stay close to the RDKit arguments, as it may otherwise be confusing. We will anyway have FP's with non-consistent arguments.